mlflow/cli.py

```diff
@@ -136,16 +137,22 @@ def run(uri, entry_point, version, param_list, docker_args, experiment_name, exp
         sys.exit(1)
 
 
-def _user_args_to_dict(user_list, flag_name='P'):
+def _user_args_to_dict(arguments, argument_type='P'):
     user_dict = {}
-    for s in user_list:
-        index = s.find("=")
-        if index == -1:
+    for arg in arguments:
+        split = arg.split('=')
+        # Docker arguments such as `t` don't require a value
+        # -> set them to True if specified
+        if len(split) == 1:
```

Comment:
can we re-introduce the check against `argument_type` so that valueless inputs are only acceptable for `argument_type=A`? We don't currently support valueless inputs for project parameters - if we wanted to do so, it would be best to create a separate PR that also updates the documentation for project parameters.